### ISSUE TITLE
只有在早上一段时间能下载

### ISSUE THREAD

**Issue by YWT-000714** [iwrqk/iwrqk#35]
只有在早上7:30到约8:30时可以下载，其他时间连不到服务器
Only in certain time pieriod(around7:30am-8:30am) would the download function be available,otherwise it would alarm for  time out to connecting the server.

**Comment by sosora7520** [comment]
这是单纯的网络和服务器问题，与本应用无关

**Comment by Kelniimir** [comment]
Indeed, most probably ISP issue, or inter-country connection issue.

Doesn't seem to be affecting many users.
Using v2.0.0+16, can download at any time. Cannot download some videos, but there's another issue about it.
